src/functions/hasServices.ts

```diff
@@ -0,0 +1,11 @@
+import { Offer, SeatMap } from 'types'
```

Comment:
💭 Could be worth adding spec like the above?

Reply:
Fair, I should! Will do

Reply:
Done (`src/functions/hasService.spec.ts`)!